.travis/spack/packages.yaml

```diff
@@ -16,6 +16,9 @@ packages:
       openmpi@1.6.5%gcc@7.2.0 arch=linux-ubuntu14-x86_64: /usr
       openmpi@1.6.5%clang@5.0.0 arch=linux-ubuntu14-x86_64: /usr
     buildable: False
+packages:
```

Comment:
one line too much ;)

@C0nsultant It was still in WIP state ;-) use at your own risk ;)